doc/nrf/includes/wifi_credentials_shell.txt

```diff
@@ -0,0 +1,20 @@
+Configuring Wi-Fi access point credentials
+==========================================
+
+This sample uses the :ref:`lib_wifi_credentials` library to manage Wi-Fi credentials.
+Before the sample can connect to a Wi-Fi network, you must configure at least one credential set.
+
+Once your device has been flashed with this sample, you can add a credential by connecting to your device's UART interface and then entering the following command:
```

Comment:
```suggestion
Once your device has been flashed with this sample, connect to your device's UART interface and add a credential using the following command:
```

Reply:
I think that it's more correct to use "add credentials using". Password and ssid counts as seperate credentials, and therefore plural form is better, no?

Reply:
Agreed, plural is better.